### ISSUE TITLE
Native new issue: extend tap area for title + comment

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/GitHawk#545]
Now only tapping the textview / textfield it will trigger, should work on the whole cell

**Comment by Sherlouk** [comment]
I'm genuinely struggling to tap part of the cell which isn't the field, it's slim margins on all sides

**Comment by BasThomas** [comment]
True! Why I labeled it `low`. Saw it when working with the simulator, that's probably why 😛

Still wouldn't hurt though!

**Comment by BasThomas** [comment]
I would actually maybe expect the title to be first responder after the view is presented even.
